cfgov/jinja2/v1/_includes/molecules/breadcrumbs.html

```diff
@@ -0,0 +1,27 @@
+{# ==========================================================================
+
+   breadcrumbs.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create breadcrumb markup when given:
+
+   items:              An array of tuples used to display nav items.
+                       format: (href, id, caption)
+
+   additional_classes: Extra classes you wish to add to the nav,
+                       space separated.
+
+   ========================================================================== #}
+
+{% macro render(breadcrumbs, additional_classes) %}
```

Comment:
I wouldn't take them out just yet. I'm pretty sure we use the additional classes to handle spacing across the different templates. I'd want to test it before removing.